frontend/hooks/useTeamIdParam.ts

```
@@ -186,6 +186,17 @@ const getUserTeams = ({
     : filterUserTeamsByRole(currentUser.teams, permittedAccessByTeamRole);
 };
 
+/** Prefer a fleet named "Workstations" (with or without emoji prefix),
+ *  otherwise fall back to the fleet with the lowest ID. */
+const preferredOrLowestIdFleet = (fleets: ITeamSummary[]) => {
+  const workstations = fleets.find(
+    (t) =>
+      t.name === "Workstations" ||
+      t.name === "\u{1F4BB} Workstations" // 💻 Workstations
```

ESLint

[error] Replace `⏎······t.name·===·"Workstations"·||⏎·····` with `·t.name·===·"Workstations"·||`

(prettier/prettier)